c_src/membrane_audio_mix_plugin/stream_format_audio_raw.h

```diff
@@ -0,0 +1,27 @@
+#include <membrane/membrane.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#define MAX_SIZE 4
+
+union Value {
+  int32_t s_val;
+  uint32_t u_val;
+};
+
+struct _StreamFormatAudioRaw {
+  uint32_t channels;
+  uint32_t sample_format;
+  uint32_t sample_rate;
+};
+typedef struct _StreamFormatAudioRaw StreamFormatAudioRaw;
```

Comment:
This name became awful after a straight-forward replacement, it deserves some creativity :wink: Orignally the Elixir struct was `Membrane.Caps.Audio.Raw` and the C-struct was derived from that.

I'd suggest making it simply `AudioRaw` (to match the Elixir struct) or `RawAudioFormat`. I'm also open to other suggestions.